src/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.Extract.cs

```diff
@@ -42,7 +42,7 @@ public static partial class ZipFile
         /// <param name="sourceArchiveFileName">The path to the archive on the file system that is to be extracted.</param>
         /// <param name="destinationDirectoryName">The path to the directory on the file system. The directory specified must not exist, but the directory that it is contained in must exist.</param>
         public static void ExtractToDirectory(string sourceArchiveFileName, string destinationDirectoryName) => 
-            ExtractToDirectory(sourceArchiveFileName, destinationDirectoryName, entryNameEncoding: null, overwrite: false);
+            ExtractToDirectory(sourceArchiveFileName, destinationDirectoryName, entryNameEncoding: null, overwriteFiles: false);
```

Comment:
The source and ref disagree. I am assuming this is correct. https://github.com/dotnet/corefx/pull/14806 cc @danmosemsft

Reply:
This one is tricky.  https://github.com/dotnet/standard/search?q=ExtractToDirectory&unscoped_q=ExtractToDirectory says that most of the platforms called it `overwrite` but the netstandard-ref calls it `overwriteFiles` (like the ref here).

So I think `overwriteFiles` is correct, but clearly there's a problem somewhere.  (cc @terrajobst)